### ISSUE TITLE
Ignore suggestion does not work

### ISSUE THREAD

**Issue by ruohki** [Clemens-E/obsidian-languagetool-plugin#53]
Hey,

for some reason "Ignore Suggestion" doesnt work for me.
When i click the button nothing happens, ive appended the electron error message that pops when i press the button.
(It doesnt even close the menu)

Running the latest version of obsidian and the plugin.

Electron console Spits out a nonsaying error:

```
Uncaught RangeError: Mark decorations may not be empty
    at t.range (app.js:1)
    at e.update [as updateF] (eval at <anonymous> (app.js:1), <anonymous>:21:14041)
    at Object.update (app.js:1)
    at e.computeSlot (app.js:1)
    at kr (app.js:1)
    at new e (app.js:1)
    at e.applyTransaction (app.js:1)
    at e.get (app.js:1)
    at e.update (app.js:1)
    at e._dispatch (app.js:1)
t.range @ app.js:1
update @ VM134:21
update @ app.js:1
(anonymous) @ app.js:1
kr @ app.js:1
e @ app.js:1
e.applyTransaction @ app.js:1
get @ app.js:1
e.update @ app.js:1
_dispatch @ app.js:1
e.dispatch @ app.js:1
i.onclick @ VM134:21
``` 

Edit: Nailed it down to be an issue with the "new" editor, it works in the legacy editor. But id like to be able to use live view

**Comment by cypherBeep** [comment]
It's still not working, I came to raise the same issue. Nothing happens when I click the ignore suggestions button.

**Comment by ruohki** [comment]
Ofc its not since the pr is not merged yet :)

**Comment by ruohki** [comment]
Fixed as per #55
